api/src/main/java/org/apache/iceberg/expressions/ExpressionUtil.java

```diff
@@ -111,7 +122,15 @@ public static boolean selectsPartitions(
 
   private static class ExpressionSanitizer
       extends ExpressionVisitors.ExpressionVisitor<Expression> {
-    private static final ExpressionSanitizer INSTANCE = new ExpressionSanitizer();
+    private final long now;
+    private final int today;
+
+    private ExpressionSanitizer() {
+      long nowMillis = System.currentTimeMillis();
+      OffsetDateTime nowDateTime = Instant.ofEpochMilli(nowMillis).atOffset(ZoneOffset.UTC);
+      this.now = nowMillis * 1000;
```

Comment:
I think it would be good to name this `nowMicros` here and in the other places to make it clearer that we're dealing with microseconds

Reply:
Done.